mockall_derive/src/mock_function.rs

```diff
@@ -1397,12 +1397,10 @@ impl<'a> ToTokens for ConcreteExpectationGuard<'a> {
         let (ei_ig, _, _) = e_generics.split_for_impl();
         let v = &self.f.privmod_vis;
         quote!(
-            ::mockall::lazy_static! {
-                #[doc(hidden)]
-                #v static ref EXPECTATIONS:
-                    ::std::sync::Mutex<Expectations #tg> =
-                    ::std::sync::Mutex::new(Expectations::new());
-            }
+            #[doc(hidden)]
```

Comment:
Can you also eliminate the lazy_static at line 1469? If so, we can eliminate lazy_static as a dependency.